extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -57,14 +61,24 @@ export class SubscriptionService {
         this.subscriptionPlatforms_.push(
             new LocalSubscriptionPlatform(
                 this.ampdoc_,
-                subscriptionPlatformConfig.paywallUrl
+                subscriptionPlatformConfig
             )
         );
       });
       resolve();
     });
   }
 
+  /**
+   * This method registers an auto initialized subcription platform with this service.
+   *
+   * @param {string} serviceId
+   * @param {!./entitlements.Entitlements} entitlements
+   */
+  registerService(serviceId, entitlements) {
```

Comment:
Hmm. The `registerService` should be with `SubscriptionPlatform`, not ready entitlements, right?

Reply:
This is the registration call for subscription platform like `amp-google-subscription`.

equivalent of this: https://github.com/ampproject/amphtml/blob/3b8c24b4dcd4aa6f8f026e9634d53d0ed8b50dd3/extensions/amp-access-laterpay/0.1/amp-access-laterpay.js#L27

Reply:
Yeah, that's the idea. But you are currently only registering the entitlements object itself?